lib/tls.js

```diff
@@ -22,9 +22,9 @@ exports.DEFAULT_CIPHERS =
 
 exports.DEFAULT_ECDH_CURVE = 'prime256v1';
 
-exports.getCiphers = internalUtil.cachedResult(() => {
-  return internalUtil.filterDuplicateStrings(binding.getSSLCiphers(), true);
-});
+exports.getCiphers = internalUtil.cachedResult(
```

Comment:
Yeah, I'll claim a combination of being tired when I did this, multi-tasking too much, and/or just being lazy. The change originally started in `tls.js` and morphed into fixing up the test. When the morphing was complete, I shouldn't have included this now-unrelated change. I'll remove it.